openmicroscopy/openmicroscopy
# Omero to imagej

To test this PR
 * Update the omero-imagej plugin
 * Open an image in imageJ
 * Draw few ROIs and add them either as overlays or to the ROIManager
 * Click on the Save ImageJ results button (in DataManager toolbar)
 * Check that The rois are saved and linked to the image.

Test 2
 * Open an image in ImageJ (not already imported)
 * Click on the Save ImageJ results button.
 * A dialog should pop up asking the user wants to import the image first. 

cc @bramalingam @pwalczysko
